syntax = "proto3";
package workflow;

import "execution_data.proto";
import "intent_data.proto";
import "meta_data.proto";

message WorkflowData {
  string id = 1;
  
  string name = 2;
  
  MetaData meta_data = 3;
  
  int32 create_timestamp = 4;

  repeated IntentData intent_data = 5;

  ExecutionData execution_data = 6;
}